### PR TITLE
Awsome day

### PR DESCRIPTION
**Pull request by mhefferan**
My blog on the Awsome day

### CODE REVIEW

File: posts/awsome_day.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+maintenance time we could achieve by moving to more intensive use of managed services.
+
+The afternoon sessions were much more useful and there are a few services I think
+we can use, and a few potential projects for the next Hack Day.
+
+Archiving old data to [Glacier](http://aws.amazon.com/glacier/) to save storage costs, we can consider DynamoDB as
+an alternative to Cassandra or RDS for some simple key/value data structures,
+we're already looking at replacing Redis with [ElastiCache](http://aws.amazon.com/elasticache/), but I think we can
+also take advantage of it to cache some common requests. We might/could be able to
+replace Chef with [OpsWork](http://aws.amazon.com/opsworks/)
```
Comment (gavinheavyside) [inline posts/awsome_day.md]: OpsWorks, not OpsWork.

replace Hosted Chef with OpsWorks *in some areas*.

File: posts/awsome_day.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+![AWSome Day banner](/assets/media/awsome_day.jpg)
+
+
+We make extensive use of AWS infrastructure [@MyDrive](https://twitter.com/_mydrive).
+We've been using them for a long time, initially using them to mirror a more traditional
+infrastructure using cloud resources, to allow portability between cloud providers.
+
+More recently we've identified that we can make considerable cost and time savings
+if we start using more of the baked in AWS Managed Services and we've begun that transition,
+for example we have already moved our database infrastructure to [RDS](http://aws.amazon.com/rds/).
```
Comment (gavinheavyside) [inline posts/awsome_day.md]: our *relational* database infrastructure

**Comment by gavinheavyside** [comment]
Other than that, :shipit:

**Comment by mhefferan** [comment]
@calonso has suggested adding a bit more technical detail so I think I could add some stuff about the features of managed services we could use, and why

File: posts/awsome_day.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+Archiving old data to [Glacier](http://aws.amazon.com/glacier/) should save us a bit
+amount on storage costs. Although the savings are small, it's so easy to set up 
+automated archiving rules on a per bucket basis that there's almost no cost to
+implement archiving. 
+
+I hadn't looked much at [DynamoDB](http://aws.amazon.com/dynamodb/) but it does seem like it might be a good addition to
+our database infrastructure. It's a simple No-SQL key, value pair store. With each
+object limited to 400KB. One nice feature is that you can configure your required
+IOPS so it's trivial to scale. It's also comfortable with updates, and supports
+a secondary index. There are certainly some small, high throughput, simple data sets
```
Comment (calonso) [inline posts/awsome_day.md]: Supports secondary indexes

**Comment by calonso** [comment]
I like it. Congratz!! This blog should take off soon!! :shipit:

**Pull request merged**